lxd/db/images.go

```diff
@@ -1091,6 +1091,36 @@ func (c *Cluster) UpdateImageUploadDate(id int, uploadedAt time.Time) error {
 	return err
 }
 
+// GetImages returns all images.
+func (c *Cluster) GetImages() (map[string][]string, error) {
+	images := make(map[string][]string) // key is fingerprint, value is list of projects
+	err := c.Transaction(func(tx *ClusterTx) error {
+		stmt := `
+    SELECT images.fingerprint, projects.name FROM images
+      LEFT JOIN images_nodes ON images.id = images_nodes.image_id
```

Comment:
Why the `images_nodes` join?

Reply:
I copied the wrong line. That should be `LEFT JOIN projects ON images.project_id = projects.id`